bert_e/workflow/gitwaterflow/branches.py

```diff
@@ -583,19 +584,18 @@ def _extract_pr_ids(self, queues):
             if len(version) == 2 and greatest_dev is None:
                 greatest_dev = version
             if len(version) == 4:
-                # we will not catch the hf pr_id later from greatest_dev
+                # we may not catch the hf pr_id later from greatest_dev
                 # so insert them now
                 for qint in queues[version][QueueIntegrationBranch]:
-                    if qint.pr_id not in prs:
-                        prs.insert(0, qint.pr_id)
+                    if qint.pr_id not in prs_hf:
+                        prs_hf.insert(0, qint.pr_id)
 
         if greatest_dev:
             for qint in queues[greatest_dev][QueueIntegrationBranch]:
-                if qint.pr_id not in prs:
+                if qint.pr_id not in prs_hf + prs:
                     prs.insert(0, qint.pr_id)
 
-        prs.sort()
-        return prs
+        return prs_hf + prs
```

Comment:
how does this works?

Reply:
this was explained offline, we're listing hotfix prs first and then the normal PR